### PR TITLE
Chore / DAO-437 

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- Re-exported testing-library/react with custom `render` function that includes a Router and internalization provider
- Changed path to the test-utils in tsconfig so utilities can be referenced as follows: ```import {screen, render} from 'test-utils'```

### CODE REVIEW

File: packages/web-app/jest.config.js [hunk 1]
```diff
@@ -4,5 +4,10 @@ module.exports = {
   collectCoverage: true,
   collectCoverageFrom: ['src/**/*.ts(x)'],
   setupFilesAfterEnv: ['<rootDir>/.jest/setup.ts'],
-  modulePaths: ['<rootDir>/src/', '<rootDir>/.jest']
-}
+  modulePaths: ['<rootDir>/src/', '<rootDir>/.jest'],
+  moduleDirectories: [
+    'node_modules', // add the directory with the test-utils.js file, for example:
```
Comment (WissenIstNacht) [inline packages/web-app/jest.config.js]: Thanks for leaving a comment. Unfortunately I don't understand it. Could you explain why `node_modules` is here?

**Review by WissenIstNacht — APPROVED** [review]
Thanks for doing this, it will certainly come in handy.

File: packages/web-app/jest.config.js (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/jest.config.js]: Setting `moduleDirectories` overrides Jest's default behavior when it's looking for modules/packages. By adding in `node_modules` in the array, we're instructing Jest to also look into that folder when it's trying to resolve a module/package. You can take a look at the corresponding documentation [here](https://jestjs.io/docs/next/configuration#moduledirectories-arraystring).

Comment (WissenIstNacht) [reply packages/web-app/jest.config.js]: Thanks a lot for the explanation 👌🏼

**Review by sepehr2github — APPROVED** [review]

**Comment by sepehr2github** [comment]
Please solve the conflicts then merge

**Pull request merged**